driver-core/src/main/com/mongodb/connection/DefaultServerMonitor.java

```diff
@@ -50,21 +49,19 @@
     private final ChangeListener<ServerDescription> serverStateListener;
     private final InternalConnectionFactory internalConnectionFactory;
     private final ConnectionPool connectionPool;
-    private final ServerSettings settings;
+    private final ServerSettings serverSettings;
     private final ServerMonitorRunnable monitor;
     private final Thread monitorThread;
     private final Lock lock = new ReentrantLock();
     private final Condition condition = lock.newCondition();
     private volatile boolean isClosed;
 
-    DefaultServerMonitor(final ServerId serverId, final ServerSettings settings,
+    DefaultServerMonitor(final ServerId serverId, final ServerSettings serverSettings, final ServerMonitorListener serverMonitorListener,
```

Comment:
Same as above: passing two serverMonitorListener parameters in, and ignoring the one in serverSettings.